features/machine/machine.feature

```diff
@@ -125,26 +125,11 @@ Feature: Machine features testing
     And I switch to cluster admin pseudo user
     And I use the "openshift-machine-api" project
     And admin ensures machine number is restored after scenario
-
+    
```

Comment:
Perhaps we do not need the change on this line ?

Reply:
yes , removed that space